create_per_gene_matrix.py

```diff
@@ -113,56 +171,51 @@ def compute_from_vp_mt(chr20: bool, overwrite: bool):
         x=True
     )
 
-    vp_mt = vp_mt.annotate_cols(
-        pop=['all', vp_mt.pop]
-    )
-    vp_mt = vp_mt.explode_cols('pop')
-
     vp_mt = vp_mt.explode_rows('vep')
+    
     vp_mt = vp_mt.transmute_rows(
         **vp_mt.vep
     )
-
+    
+    vp_mt = vp_mt.explode_rows('csq')
+    
+    vp_mt = vp_mt.checkpoint('gs://gnomad-tmp/compound_hets/chet_per_gene{}.1.mt'.format(
+        '.chr20' if chr20 else ''
+    ), overwrite=True)
+    
     def get_grouped_phase_agg():
         return hl.agg.group_by(
             hl.case()
-                .when(~vp_mt.is_singleton_vp & (vp_mt.phase_info[vp_mt.pop].em.adj.p_chet > CHET_THRESHOLD), 1)
-                .when(~vp_mt.is_singleton_vp & (vp_mt.phase_info[vp_mt.pop].em.adj.p_chet < SAME_HAP_THRESHOLD), 2)
+                .when(~vp_mt.is_singleton_vp & (vp_mt.phase_info['all'].em.adj.p_chet >= CHET_THRESHOLD), 1)
+                .when(~vp_mt.is_singleton_vp & (vp_mt.phase_info['all'].em.adj.p_chet <= SAME_HAP_THRESHOLD), 2)
                 .default(3)
             ,
-            hl.agg.min(vp_mt.csq)
+            hl.agg.collect_as_set(vp_mt.csq)
         )
 
     vp_mt = vp_mt.group_rows_by(
         'gene_id',
         'gene_symbol'
     ).aggregate(
-        all=hl.agg.filter(
-            vp_mt.x &
-            hl.if_else(
-                vp_mt.pop == 'all',
-                hl.is_defined(vp_mt.popmax_af) &
-                (vp_mt.popmax_af <= MAX_FREQ),
-                vp_mt.pop_af[vp_mt.pop] <= MAX_FREQ
-            ),
-            get_grouped_phase_agg()
-        ),
-        af_le_0_001=hl.agg.filter(
-            hl.if_else(
-                vp_mt.pop == 'all',
-                hl.is_defined(vp_mt.popmax_af) &
-                (vp_mt.popmax_af <= 0.001),
-                vp_mt.pop_af[vp_mt.pop] <= 0.001
+        **{
+            f"af_le_{af}": hl.agg.filter(
+                    hl.is_defined(vp_mt.popmax_af) &
+                    (vp_mt.popmax_af <= af) &
+                    vp_mt.x,
+                get_grouped_phase_agg()
             )
-            & vp_mt.x,
-            get_grouped_phase_agg()
-        )
+            for af in ALLELE_FREQUENCY_CUTOFFS
+        }
     )
 
     vp_mt = vp_mt.checkpoint('gs://gnomad-tmp/compound_hets/chet_per_gene{}.2.mt'.format(
```

Comment:
There will be a lot less rows in this table since it is now by gene, might want to repartition